src/util/ajax.js

```diff
@@ -39,7 +39,29 @@ if (typeof Object.freeze == 'function') {
  * @typedef {Object} RequestParameters
  * @property {string} url The URL to be requested.
  * @property {Object} headers The headers to be sent with the request.
+ * @property {string} method Request method `'GET' | 'POST' | 'PUT'`.
+ * @property {string} body Request body.
+ * @property {string} type Response body type to be returned `'string' | 'json' | 'arrayBuffer'`.
  * @property {string} credentials `'same-origin'|'include'` Use 'include' to send cookies with cross-origin requests.
+ * @property {boolean} collectResourceTiming If true, Resource Timing API information will be collected for these transformed requests and returned in a resourceTiming property of relevant data events.
+ * @example
+ * // transformRequest used to modify requests that begin with `http://myHost`
+ * transformRequest: function(url, resourceType) {
+ *  if(resourceType === 'Source' && url.startsWith('http://myHost')) {
```

Comment:
We could change it to:

```suggestion
 *  if (resourceType === 'Source' && url.indexOf('http://myHost') > -1) {
```

Reply:
seems reasonable to me!